benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -53502,7 +53502,7 @@ exports[`Render Summary for rows-with-scores.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={"0&#x2008;&#x2007;&#x2007;&#x2007;&#x2007;"}
-         title="avg: 0, max: 0, median: 0, min: 0, stdev: 0"
+         title="avg: 0, max: 0, median: 0, min: 0, stdev: 0.000"
```

Comment:
Sorry for re-iterating over this again but could you outline the rules for padding stdev again? 
We only do it for some column types, right?

Reply:
If significant digits are given, we always use them. If there are none given, and this is a measure column, we use a default of 3 significant digits. If it is a count column, we use a default of 2 decimal digits.

Reply:
Okay, gotcha. In this case, the column has a `number_of_significant_digits` of 3, so i understand the new result `0.000` would be correct?
Again, this (and all other currently failing snapshots) appear in `rows-with-scores.html` and `test.2015-03-03_1613.results.predicateAnalysis.custom-score.html´, so i don't know how much manual creation might have influenced this result.

Reply:
Are you sure this happens with `test.2015-03-03_1613.results.predicateAnalysis.custom-score.html`? I saw only `0.000` as stdev in it when I looked at it in my browser. If there are wrong values, I guess the easiest way to show me would be a comment for the respective line on GitHub.

For `rows-with-scores.html`, as usual please just fix the JSON.